src/common/query/src/signature.rs

```diff
@@ -118,14 +118,14 @@ impl From<TypeSignature> for DfTypeSignature {
     fn from(type_signature: TypeSignature) -> DfTypeSignature {
         match type_signature {
             TypeSignature::Variadic(types) => {
-                DfTypeSignature::Variadic(concret_types_to_arrow_types(types))
+                DfTypeSignature::Variadic(concert_types_to_arrow_types(types))
```

Comment:
It should be `concrete` instead of `concert`

Reply:
Thank you, updated